oss_formatter.go

```diff
@@ -50,12 +51,16 @@ func (f *OSSFormatter) Format(entry *logrus.Entry) ([]byte, error) {
 	b.WriteString(fmt.Sprintf(" msg='''%s'''", entry.Message))
 	b.WriteByte(' ')
 	for key, value := range entry.Data {
-		appendKeyValue(b, key, value)
+		appendKeyValue(b, replaceInvalidChars(key), value)
```

Comment:
It will affect all the services. Is it better that we remove all dots from service side?

Reply:
Hmm. No other users of this formatter except rec-async (we wrote this formatter only after we decided to separate log from context). Anyway, if the keys contain `.`, e.g. `grpc.time_ms`, it will not show on `logs-es`. So no breaking change to any scripts.

Reply:
IC